spec.bs

```diff
@@ -1502,7 +1524,70 @@ The <a spec="turtledove">validate and convert auction ad config</a> steps are
 modified to add the following steps just before the last step ("Return
 <var ignore>auctionConfig</var>"), renumbering the later step as appropriate:
 <div algorithm="protected-audience-validate-config-monkey-patch">
-7. If |config|[{{AuctionAdConfig/privateAggregationConfig}}] [=map/exists=]:
+31. If |config|["<code>{{AuctionAdConfig/auctionReportBuyerKeys}}</code>"]
+    [=map/exists=]:
+    1. Let |interestGroupBuyers| be |auctionConfig|'s <a spec="turtledove"
+        for="auction config">interest group buyers</a>.
+    1. If |interestGroupBuyers| is null, set |interestGroupBuyers| to a new
+        [=list=].
```

Comment:
it seems these two steps are identical to check "If |config|["<code>{{AuctionAdConfig/auctionReportBuyerKeys}}</code>"]
    [=map/exists=] and |auctionConfig|'s <a spec="turtledove" for="auction config">interest group buyers</a> is not null:".

But the implementation seems not checking this, but just simply copying {{AuctionAdConfig/auctionReportBuyerKeys}}